lib/libutee/sub.mk

```diff
@@ -12,5 +12,32 @@ srcs-y += tee_api_objects.c
 srcs-y += tee_api_operations.c
 srcs-y += tee_api_se.c
 srcs-y += tee_api_panic.c
+srcs-y += utf8.c
+srcs-y += image.c
+srcs-y += image_png.c
+srcs-y += font.c
+
+# Need to locate font.h from generated sources
+incdirs-y += .
+
+gensrcs-y += default_bold
+produce-additional-default_bold = default_bold.h
+produce-default_bold = default_bold.c
+depends-default_bold = scripts/render_font.py \
+		%this-dir%/amble/Amble-Bold.ttf
+recipe-default_bold = scripts/render_font.py \
+		--font_file %this-dir%/amble/Amble-Bold.ttf \
+		--font_size 20 --font_name default_bold \
+		--out_dir %out-dir%
```

Comment:
So if I understand correctly, `%out-dir%` is replaced by the full path to the output directory that corresponds to the current source sub-directory.
Could we define a normal macro instead? Such as `$(out-subdir)`. In `mk/subdir.mk` I suppose. This would be more consistent with $(out-dir) and so on.